### ISSUE TITLE
prohibited aria-label usage

### ISSUE THREAD

**Issue by Trott** [tomusborne/generatepress#555]
### Description

The following line is a bug and should be commented out/removed:

https://github.com/tomusborne/generatepress/blob/a4b04b12e1ad6c88c6109f23f196d61895f3d74b/inc/class-html-attributes.php#L290 

This attribute gets  applied to a `<header>` element.  From https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/banner_role:

> By default, the HTML's &lt;header> element has an identical meaning to the banner landmark, unless it is a descendant of &lt;aside>, &lt;article>, &lt;main>, &lt;nav>, or &lt;section>, at which point &lt;header> exposes a generic role, and not the equivalent of the site-wide banner.

Since this `<header>` element is a child of an `<article>` element, it has the generic role.

The generic role cannot have an `aria-label` attribute. From https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/generic_role:

> Because the generic role is nameless, the aria-labelledby and aria-label attributes are prohibited.

### Steps to reproduce

1. Run a site with GeneratePress.

#### Actual behavior

`<header>` element contained within an `<article>` element has an `aria-label` attribute.

#### Expected behavior

`<header>` element contained within an `<article>` element does not have an `aria-label` attribute.

---

#### References

* https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html
* https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA5
* https://alfa.siteimprove.com/rules/sia-r18

**Comment by JeanPaiva** [comment]
Thank you @Trott we will look into this.
